src/components/SearchResults.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import { Redirect } from 'react-router-dom'
+
+import MovieInfo from './MovieInfo'
+import {showMovieDetails, getMoreResults} from '../redux/actions'
+
+class SearchResults extends React.Component {
```

Comment:
Can you please use https://material-ui.com/getting-started/installation/ instead of the raw html elements?